packages/apps/fortune/recording-oracle/src/modules/storage/storage.service.ts

```diff
@@ -87,7 +87,7 @@ export class StorageService {
     }
 
     let fileToUpload = JSON.stringify(solutions);
-    if (this.serverConfig.pgpEncrypt as boolean) {
+    if (!(this.serverConfig.pgpEncrypt === 'false')) {
```

Comment:
didn't it work before? In any case I suggest to do it like 
```suggestion
    if (this.serverConfig.pgpEncrypt === 'true') {
```
Since the default value should be false, I would only enable encryption if the value of the variable is strictly `true`

Reply:
Before was always true as the boolean takes any value different from 0 or empty as true. I'll use the suggestion